runhouse/resources/hardware/utils.py

```diff
@@ -395,12 +395,12 @@ def parse_filters(since: str, cluster_status: Union[str, ClusterStatus]):
 def get_clusters_from_den(cluster_filters: dict, force: bool):
     get_clusters_params = {"resource_type": "cluster", "folder": rns_client.username}
 
-    if (
-        "cluster_status" in cluster_filters
-        and cluster_filters["cluster_status"] == ClusterStatus.TERMINATED
+    if "cluster_status" in cluster_filters and any(
```

Comment:
confused about this, seems like we're miixsing cluster status and daemon status? what's the full context?